ui/src/chat/ChatMessage/Author.tsx

```diff
@@ -33,19 +28,15 @@ export default function Author({
   const location = useLocation();
   const { didCopy, doCopy } = useCopy(ship);
   const modalNavigate = useModalNavigate();
-  const prettyTime = date ? makePrettyTime(date) : undefined;
-  const prettyDayAndTime = date ? makePrettyDayAndTime(date) : undefined;
-  const prettyDayAndDateAndTime = date
-    ? makePrettyDayAndDateAndTime(date)
-    : undefined;
+  const timeDisplay = date ? makePrettyDayAndDateAndTime(date) : undefined;
 
   const handleProfileClick = () => {
     modalNavigate(`/profile/${ship}`, {
       state: { backgroundLocation: location },
     });
   };
 
-  if (!date) {
+  if (!timeDisplay) {
```

Comment:
Why do we need to update this?

Reply:
mostly just didn't want to do ?. everywhere were referencing time display. but if you mean the time display stuff itself, it's because the screen reader was reading the bullet lol.